src/i18n/strings/en_EN.json

```diff
@@ -2478,6 +2478,7 @@
     "Topic (optional)": "Topic (optional)",
     "Room visibility": "Room visibility",
     "Private room (invite only)": "Private room (invite only)",
+    "Anyone can knock to join" : "Anyone can knock to join",
```

Comment:
You probably need to run the `yarn i18n` script so that all the new translatable strings are added here